lib/mpl_toolkits/axes_grid1/axes_divider.py

```diff
@@ -48,6 +49,7 @@ def __init__(self, fig, pos, horizontal, vertical,
         self._horizontal = horizontal
         self._vertical = vertical
         self._anchor = anchor
```

Comment:
Can drop this line now?

Reply:
Won't some code quality tools complain if it is defined outside of `__init__`? That was the reason anyway.